packages/kbn-es-query/src/kuery/functions/is.js

```diff
@@ -88,6 +90,27 @@ export function toElasticsearchQuery(node, indexPattern = null, config = {}) {
   }
 
   const queries = fields.reduce((accumulator, field) => {
+    const wrapWithNestedQuery = (query) => {
+      // Wildcards can easily include nested and non-nested fields. There isn't a good way to let
+      // users handle this themselves so we automatically add nested queries in this scenario.
```

Comment:
Maybe we should consider adding an advanced setting so that users can choose if they want to include nested queries with wildcards.

Reply:
What would the use case be for excluding nested fields? I think generally when using wildcards a user is trying to cast a wide net, essentially saying "find this value in any field that matches this pattern". I don't see why a field should be left out of that search just because it's nested.